receiver/k8sclusterreceiver/metadata.yaml

```diff
@@ -274,6 +278,14 @@ metrics:
     unit: ""
     gauge:
       value_type: int
+  k8s.container.status_last_terminated_reason:
+    enabled: false
+    description: Last terminated reason of container. The unit is always 1.
```

Comment:
```
The unit is always 1.
```
I believe, from the other code being added, this should instead say the **value** will always be 1. Am I correct or do you mean something else here?